tools/pipelines/templates/build-docker-service.yml

```diff
@@ -93,7 +93,7 @@ parameters:
 # Any additional build arguments to pass to the docker build command.
 - name: additionalBuildArguments
   type: string
-  default: ""
+  default: "--build-context root=."
```

Comment:
Can we document this default?

Reply:
I'm actually very tempted to revert all the changes[8728c12](https://github.com/microsoft/FluidFramework/pull/18445/commits/8728c12da1f7a4a662b12102b287a2e6649e727b), I haven't been able to make them work yet.

Reply:
Reverted this.